python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -646,6 +646,14 @@ def applyOverrides(cls, config):
         config.isr.doFringe = False
 
 
+class DarkCombineTask(CalibCombineTask):
+    """Task to combine dark images"""
+    def run(*args, **kwargs):
+        combined = CalibCombineTask.run(*args, **kwargs)
+        combined.getMetadata().set("EXPOSURE", 1.0)
+        combined.getMetadata().set("EXPTIME", 1.0)
```

Comment:
Why force an exposure time of 1?  I don't see where the dark counts are normalized.  Ideally, this should come from the calib object right?